INTRODUCTION.md

```diff
@@ -1288,6 +1292,26 @@ The latest stored offset will be automatically committed every
 Broker based consumer groups (requires Apache Kafka broker >=0.9) are supported,
 see KafkaConsumer in rdkafka.h or rdkafkacpp.h
 
+##### Static consumer groups
+
+By default Kafka consumers are rebalanced each time a new consumer joins
+the group or an existing member leaves. This is what is known as a dynamic
+membership. Apache Kafka >= 2.3.0 introduces static membership.
+Unlike dynamic membership, static members can leave and rejoin a group
+within the `session.timeout.ms` without triggering a rebalance retaining
+their existing partitions assignment.
```

Comment:
I would add something like: "Consumers with `group.instance.id` set will not send a leave group request on close - session timeout is the only mechanism that will trigger the group rebalance in this case" (as a n00b, this wasn't at all clear to me without digging deep into the kip).